client/src/testing/components/Testing.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FC } from 'react';
+import { sendUpdateMessage, sendLoginMessage } from '../../websocket/actions'
```

Comment:
This won't work since you need to put it in a mapDispatchToProps. You'll have to create a TestingContainer file to do that

Reply:
Yeah if just copy the logic from `client\src\about\containers\AboutContainer.tsx` for these two messages and it should work.

Reply:
Okay yeah that's what I tried and it still doesn't work 🙃. we can all just look at it tomorrow

Reply:
I changed the Routing container to use the TestingContainer rather than Testing page and the actions in the websocket are now being called!